Javascript(ES5)/README.md

```diff
@@ -283,7 +356,46 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
     }());
     ```
 
-  - Never declare a function in a non-function block (if, while, etc). Assign the function to a variable instead. Browsers will allow you to do it, but they all interpret it differently, which is bad news bears.
+  - Use named function expressions instead of function declarations.
+
+    eslint: [`func-style`](http://eslint.org/docs/rules/func-style)
+
+    > Why? Function declarations are hoisted, which means that it’s easy - too easy - to reference the function before it is defined in the file. This harms readability and maintainability. If you find that a function’s definition is large or complex enough that it is interfering with understanding the rest of the file, then perhaps it’s time to extract it to its own module! Don’t forget to name the expression - anonymous functions can make it harder to locate the problem in an Error's call stack. ([Discussion](https://github.com/airbnb/javascript/issues/794))
+
+    ```javascript
+    // bad
+    function foo() {
+      // ...
+    }
+
+    // bad
+    var foo = function () {
+      // ...
+    };
+
+    // good
+    var foo = function bar() {
```

Comment:
What do you mean? Are you referring to the name `bar` in the function name?

I believe it is valid, and based on the guides that we have, recommended (see the discussion above).

I found [this recommendation](http://bguiz.github.io/js-standards/es5/functions/#avoid-anonymous-functions:904d9cf85d985550c869e6d9741448cc) that explains this case for ES5, and the [ES5 specification](https://www.ecma-international.org/ecma-262/5.1/#sec-13) seems to be returning the function itself on a function declaration.

If I misunderstood your question, please let me know.